src/Components/Components/ref/Directory.Build.props

```diff
@@ -1,9 +0,0 @@
-<Project>
-
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
-
-  <ItemGroup>
```

Comment:
Why were we able to delete this file?

Reply:
Because the project file was fixed to include the *.Manual.cs file.

More generally, as I mentioned in the PR's description, I got rid of numerous one-off workarounds by centralizing the logic.